.editorconfig

```diff
@@ -20,8 +17,5 @@ insert_final_newline = false
 [Makefile]
 indent_style = tab
 
-[*.{js,json,coffee,less,sass,scss,css,rb}]
-indent_size = 2
```

Comment:
@morewry Yup, it will still work that way, as long as you have an editorconfig plugin. It's not that it's redundant with my or someone else's editor. It's that this setting is already set a few lines above.